packages/tpetra/core/src/Tpetra_CrsGraph_decl.hpp

```diff
@@ -590,9 +590,30 @@ namespace Tpetra {
       return copier_type::clone (*this, node2, params);
     }
 
+
     //! Destructor.
     virtual ~CrsGraph () = default;
 
+
+    /// \brief Swaps the data from *this with the data and maps from graph
+    ///
+    /// \param graph [in/out] a crsGraph
+    void swap(CrsGraph<LocalOrdinal, GlobalOrdinal, Node> & graph);
+
+
+    /// \brief True if and only if \c CrsGraph is exactly identical to this CrsGraph
+    ///
+    /// This performs _exact_ matches on objects with in the graphs. That is,
+    /// internal data structures such as arrays must match exactly in both
+    /// content and order. This is not performing any sort of isomorphic
+    /// search.
+    ///
+    /// \param graph [in] a crsGraph to compare against this one.
+    ///
+    /// WCMCLEN WORK_IN_PROGRESS
```

Comment:
Please remove WIP comment

Reply:
doh... thought I got all of those.  Removing it now.